src/caja-navigation-action.c

```diff
@@ -58,33 +54,7 @@ enum
     PROP_WINDOW
 };
 
-GType
-caja_navigation_action_get_type (void)
```

Comment:
The prototype for this function is still in the header file, and it's used in a type macro? what does this function do anyway?

Reply:
okay I should have read the earlier comments, but still there is a prototype for this function. If the function is generated by the macro, why do we need a prototype for it then?